rolling-shutter/keyperimpl/gnosis/newslot.go

```diff
@@ -219,7 +219,8 @@ func getTxPointer(ctx context.Context, db *pgxpool.Pool, eon int64, maxTxPointer
 			Int64("tx-pointer", txPointer).
 			Int64("tx-pointer-age", txPointerAge).
 			Msg("outdated tx pointer")
-		txPointer, err = gnosisKeyperDB.GetTransactionSubmittedEventCount(ctx, eon)
+		txPointerInt64, err := gnosisKeyperDB.GetTransactionSubmittedEventCount(ctx, eon)
```

Comment:
the naming looks weird. Why is it called with `Int64` suffix, but then gets casted into int64